### ISSUE TITLE
🐛 BUG: `wrangler pages publish .` relative dir picking up paths incorrectly

### ISSUE THREAD

**Issue by GregBrimble** [cloudflare/wrangler2#938]
### What version of `Wrangler` are you using?

2.0.1

### What operating system are you using?

all

### Describe the Bug

`wrangler pages publish .` results in incorrect file paths being uploaded (vs. `wrangler pages publish public`)

**Comment by MitchellCash** [comment]
This bug is hitting me as well.

If I have a website directory like this:

```
.
├─ README.md
├─ dist
   ├─ my-website
      ├─ index.html
      ├─ assets
         ├─ cat.png
```

**Example 1 ❌**

Deploy from path: `/<path>/<to-project>`
Run command: `wrangler pages publish dist/my-website`
Directory list on Cloudflare:

```
├─ index.html
├─ my-website
   ├─ assets
      ├─ cat.png
```


**Example 2 ❌**

Deploy from path: `/<path>/<to-project>/dist/my-website`
Run command: `wrangler pages publish .`
Directory list on Cloudflare:

```
├─ index.html
├─ cat.png
```


**Example 3 ✅**

Deploy from path: `/<path>/<to-project>/dist`
Run command: `wrangler pages publish my-website`
Directory list on Cloudflare:

```
├─ index.html
├─ assets
   ├─ cat.png
```

Any more questions/examples let me know! Although it's got a workaround in **Example 3**, I would love to see this fix to avoid those extra steps in the CI just to change path to deploy 😄

**Comment by GregBrimble** [comment]
Thank you for that additional information and test-cases. That'll really help us get this right! 👍

**Comment by paulobrien** [comment]
Came across this too, thought I was going crazy!

**Comment by GregBrimble** [comment]
This has been fixed in [2.0.15](https://github.com/cloudflare/wrangler2/pull/1284).
